duo_client/admin.py

```diff
@@ -2676,6 +2750,42 @@ def delete_u2ftoken(self, registration_id):
         response = self.json_api_call('DELETE', path, {})
         return response
 
+    def get_webauthncredential_by_id(self, webauthnkey):
+        """ Returns webauthn credentials specified by webauthnkey.
+
+            Params:
+                webauthnkey (str): The registration id of the
+                    webauthn credentials to fetch.
+
+            Returns:
+                Returns a webauthn credentials dict.
+
+            Notes:
+                Raises RuntimeError on error.
+        """
+        webauthnkey = \
+            six.moves.urllib.parse.quote_plus(str(webauthnkey))
+        path = '/admin/v1/webauthn-credentials/' + webauthnkey
+        response = self.json_api_call('GET', path, {})
+        return response
+
+    def delete_webauthncredential(self, webauthnkey):
+        """ Deletes a webauthn credentials. If the webauthn credentials is already
+            deleted, does nothing.
+
+            Params:
+                webauthnkey (str): The registration id of the
+                    webauthn credentials.
+
+            Notes:
+                Raises RuntimeError on error.
+        """
+        webauthnkey = \
+            six.moves.urllib.parse.quote_plus(str(webauthnkey))
+        path = '/admin/v1/webauthn-credentials/' + webauthnkey
```

Comment:
This should also call `/admin/v1/webauthncredentials`. We changed this recently.